OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -136,7 +136,16 @@ public void BuildMusicTable()
 			{
 				var song = s;
```

Comment:
Would you remind removing this long-redundant compiler workaround while you're here? Assign to `song` directly in the loop.

Reply:
Yes, I saw that in several other places (players?, maps?) as well and I was wondering why it's made like that. I'll change it.

Reply:
See https://blogs.msdn.microsoft.com/ericlippert/2009/11/12/closing-over-the-loop-variable-considered-harmful/ for context